app/models/service_ansible_playbook.rb

```diff
@@ -3,7 +3,11 @@ class ServiceAnsiblePlaybook < ServiceGeneric
 
   # A chance for taking options from automate script to override options from a service dialog
   def preprocess(action, add_options = {})
-    _log.info("Override with new options: #{add_options}") unless add_options.blank?
+    unless add_options.blank?
+      _log.info("Override with new options:")
+      $log.log_hashes(add_options)
```

Comment:
Feel strange we have to mix the use of `_log` and `$log`. Can we make `_log` to support `#log_hashes`?